pkg/cosign/verify.go

```diff
@@ -332,6 +347,10 @@ func getSubjectAlternateNames(cert *x509.Certificate) []string {
 	for _, uri := range cert.URIs {
 		sans = append(sans, uri.String())
 	}
+	otherName, _ := UnmarshalSANS(cert.Extensions)
```

Comment:
Did you mean to swallow the error? If so, can you add a comment?

Reply:
Yes I did - If there's no OtherName, it'll throw an error. Left a comment